platform/onebox/backup_and_restore/ceph/export.sh

```diff
@@ -35,28 +35,42 @@ function initialize_variables() {
   export AWS_ACCESS_KEY_ID=$(cat $CREDENTIALS_FILE | jq -r 'select(.AWS_ACCESS_KEY_ID != null) | .AWS_ACCESS_KEY_ID')
   export AWS_SECRET_ACCESS_KEY=$(cat $CREDENTIALS_FILE | jq -r 'select(.AWS_SECRET_ACCESS_KEY != null) | .AWS_SECRET_ACCESS_KEY')
   readonly FOLDER=${BASE_PATH}/ceph/
+  mkdir -p ${FOLDER}
 }
 
 function list_buckets() {
   temp_buckets=$(aws s3 --endpoint-url $AWS_ENDPOINT --no-verify-ssl ls)
   readonly BUCKETS=${temp_buckets}
 }
 
-function download_blobs() {
+function get_cors_policy() {
+  BUCKET_NAME=${1}
+  aws --endpoint-url $AWS_ENDPOINT --no-verify-ssl s3api get-bucket-cors --bucket ${BUCKET_NAME} > ${FOLDER}${BUCKET_NAME}-cors.json
+}
+
+function download_blob() {
+  BUCKET_NAME=${1}
+  echo "$green $(date) Starting sync of object storage to local disk for bucket ${BUCKET_NAME} $default"
+  mkdir -p ${FOLDER}${BUCKET_NAME}
+  aws s3 --endpoint-url $AWS_ENDPOINT --no-verify-ssl sync s3://${BUCKET_NAME} ${FOLDER}${BUCKET_NAME} --delete
```

Comment:
are we checking anywhere if "aws s3" CLI is installed ?

Reply:
yes, in pre-req.